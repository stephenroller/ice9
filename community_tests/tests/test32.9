# mod tests

write 9 % 7;
write -9 % 7;
write -9 % -7;
write 9 % -7;
write 9 % 3;
