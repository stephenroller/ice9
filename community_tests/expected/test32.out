Loading file: generated/test32.tm
2 
-2 
-2 
2 
0 
Number of instructions executed = 78
